subset/switches/src/main/java/switchtest/SwitchInterrogator.java

```diff
@@ -478,6 +535,15 @@ private void validateTests() {
         login_report += "\nswitch.port.duplex=false";
       }
 
+      if (switchSupportsPoe) {
+        if (Integer.parseInt(current_max_power) > Integer.parseInt(current_power)
+            && current_PoE_admin.equals("Enabled")) {
+          login_report += "\nswitch.port.poe=true";
```

Comment:
generally much better to always end with a newline -- it's just much more consistent. I see you have this other places, but -- in general, if the code looks like this it's doing something wrong (because nothing should ever be right after "true").

Reply:
fixed validation output